payments_service/urls.py

```diff
@@ -1,15 +1,19 @@
 from django.conf.urls import include, url
 
 
-urlpatterns = [
+api_patterns = [
     url(r'^auth/',
         include('payments_service.auth.urls', namespace='auth')),
     url(r'^braintree/',
         include('payments_service.braintree.urls', namespace='braintree')),
-    url(r'',
+    url(r'^$',
```

Comment:
theoretically, this might have more status-y endpoints which is why I left it open as ``r''``. I'm not sure if that's a good idea or not. Maybe we should make a separate ``r'status/'`` prefix *but* still alias the overall system status page to the index URL.